src/app/window-handler.ts

```diff
@@ -785,9 +785,7 @@ export class WindowHandler {
                 if (displayId === element.id.toString()) {
                     if (isWindowsOS) {
                         logger.info(`window-handler: element:`, element);
-                        const winX: string = element.bounds.x.toString();
-                        const winY: string = element.bounds.y.toString();
-                        this.execCmd(this.screenShareIndicatorFrameUtil, [ winX, winY ]);
+                        this.execCmd(this.screenShareIndicatorFrameUtil, [ displayId ]);
```

Comment:
@mattias-symphony There is still an open issue on the `electron` https://github.com/electron/electron/issues/15111 where the `displayId` will be empty in the sources object on `Windows 7`

Reply:
So what should we do, any suggestion?

Reply:
@mattias-symphony Can we use this https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-monitorfrompoint API to get the display monitor using `x & y`

Reply:
Actually, this whole block of code was already wrapped in `if (displayId !== '')` so it already did not worked on windows 7 - so I think this is fine, we have no regression due to the fix (but possibly another issue, for win7 only, which we might want to handle as a separate issue)

Reply:
> @mattias-symphony Can we use this https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-monitorfrompoint API to get the display monitor using `x & y`

No, x & y will not work because of DPI issues, we can't map accurately from electron x y to windows x y in all scenarios. Using display id is the more stable way to go. Perhaps we could use x y for windows 7 only, if we decide to fix that issue (which I think should be considered separate from this)